README.md

```diff
@@ -8,6 +8,24 @@ TODO: add a list of features
 ### Can I use this on a Vanilla server?
 Yeah sure. FoxClient is 100% client side and doesn't require anything to be installed on the server.
 
+### How can i use the latest Features?
```

Comment:
This section is empty

Reply:
Actually, I'd just change the name of this section to "Build instructions" or something